samples/embedded-auth-with-sdk/src/main/java/com/okta/spring/example/controllers/LoginController.java

```diff
@@ -136,6 +146,65 @@ public ModelAndView selectAuthenticator(final @RequestParam("authenticator-type"
 
         ProceedContext proceedContext = Util.getProceedContextFromSession(session);
 
+        if (authenticatorType != null && authenticatorType.equals("Security Key or Biometric")) {
```

Comment:
Can we make this decision based off the authenticator `type` or `key`, rather than `displayName`?

See https://github.com/okta/okta-idx-java/pull/251/files#diff-0ca4e5f951f97af0dec995ec574fff6f8384b2cbcb38911f1cc4e5cef986b391R159